conftest.py

```diff
@@ -140,6 +140,10 @@ def get_outcome_and_reason(report, markers, call):
         return report.outcome, None  # passed, failed, skipped
 
 
+def clean_docstring(docstr):
+    return " ".join([word for word in docstr.replace("\n", " ").strip().split(" ") if word != ""])
```

Comment:
An example of what this is doing might be useful.

Reply:
fixed